.github/workflows/ci.yml

```
@@ -0,0 +1,26 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches:
+      - '**'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Install dependencies
+        run: npm install
```

```suggestion
        run: npm ci
```
Я б замість npm install радив використати npm ci. Таким чином ми гарантуємо, що в workflow будуть завантажені ідентичні пакети, які ми використовували при розробці чи дебагінгу.
Ознайомся з [What is the difference between "npm install" and "npm ci"?](https://stackoverflow.com/questions/52499617/what-is-the-difference-between-npm-install-and-npm-ci) на StackOverflow чи [npm docs](https://docs.npmjs.com/cli/v11/commands/npm-ci).
Якщо коротко: "Use npm ci if you need a deterministic, repeatable build. For example during continuous integration, automated jobs, etc. and when installing dependencies for the first time, instead of npm install."